css/main.css

```diff
@@ -1470,7 +1418,7 @@ nav {
 
   /* TOP SECTION */
   .top {
-    height: 1700px;
+    height: 1350px;
```

Comment:
Is there any way in which this can be changed to a max-height?  Specifying height properties doesn't usually go well with mobile devices at all...